templates/qpidd.conf.erb

```diff
@@ -30,6 +30,7 @@
 <% unless [nil, :undefined, :undef, ''].include?(scope['qpid::acl_content']) -%>
 acl-file=<%= scope['qpid::acl_file'] %>
 <% end %>
+data-dir=<%= scope['qpid::data_dir'] %>
```

Comment:
If we are going to manage the data directory, we should specify it in the configuration for consistency across installs.

Reply:
:+1: this will be necessary if the user specifies some non-default directory